README.md

```diff
@@ -24,53 +24,61 @@ We would love to hear from the larger community: please provide feedback proacti
 Please refer to the [contribution guide](https://github.com/newrelic-forks/opentelemetry-java/blob/master/CONTRIBUTING.md)
 on how to setup and contribute!
 
-## Plan
-
-[Please review the roadmap here](https://medium.com/opentracing/a-roadmap-to-convergence-b074e5815289).
-
-In the coming months [we plan to merge the OpenCensus and OpenTracing
-projects](https://medium.com/opentracing/merging-opentracing-and-opencensus-f0fe9c7ca6f0).
-The technical committee will drive the merge effort. We’ve identified areas that
-require deeper discussion and areas that merely require alignment around
-terminology and usage patterns.
-
-We have a three-step plan for the project merge: 
-
-1. spike of merged interfaces,
-2. beta release of new project binaries, and
-3. kicking off the work towards a 1.0 release.
-
-### Spike of merged interfaces
-
-Spike API merge will happen in a separate repository, focused on Java
-specifically. The main goal of the spike is to make sure that we have a clear
-path forward, and that there will be no unforeseen technical issues blocking the
-merge of the projects (while staying true to our  declared goals for the merge).
-
-As a result of the spike we plan to produce:
-
-- Alpha version of a merged interface in new repository.
-- Rough port of OpenCensus implementation as an implementation of choice for
-  this API.
-- Rough OpenTracing bridge to new interface.
-- Supplemental documentation and design documents
-
-We expect this spike will take us a few weeks.
-
-### Beta release of a new project
-
-Once we have cleared out the path - we plan to initiate a transition of active
-contribution and discussions from OpenCensus and OpenTracing to the new project.
-We will
-
-- Clean up OpenCensus into official SDK of new project
-- Release an official OpenTracing bridge to new Interface
-
-We will minimize the duration of this transition so that users can switch to the
-new API as seamlessly as possible, and contributors can quickly ensure that
-their work is compatible with future versions. We will also encourage all
-contributors to start working in the new project once the merger announced. So
-there will be no time of duplicative contributions.
+## Snapshots
+
+Snapshots based out the `master` branch are available for `opentelemetry-api`, `opentelemetry-sdk` and the rest of the artifacts:
+
+### Maven
+
+```xml
+  <repositories>
+    <repository>
+      <id>oss.sonatype.org-snapshot</id>
+      <url>https://oss.jfrog.org/artifactory/oss-snapshot-local</url>
+    </repository>
+  </repositories>
+
+  <dependencies>
+    <dependency>
+      <groupId>io.opentelemetry</groupId>
+      <artifactId>opentelemetry-sdk</artifactId>
+      <version>0.1.0-SNAPSHOT</version>
+    </dependency>
+  </dependencies>
+```
+
+### Gradle
+
+```groovy
+repositories {
+	maven { url 'https://oss.jfrog.org/artifactory/oss-snapshot-local' }
+}
+
+dependencies {
+	compile('io.opentelemetry:opentelemetry-sdk:0.1.0-SNAPSHOT')
```

Comment:
```suggestion
	compile('io.opentelemetry:opentelemetry-api:0.1.0-SNAPSHOT')
```

Reply:
Should we list both dependencies?

Reply:
We can mention that libraries probably only need to depend on APIs, while applications need both. See https://github.com/census-instrumentation/opencensus-java#opencensus-quickstart-for-libraries and https://github.com/census-instrumentation/opencensus-java#opencensus-quickstart-for-applications.